### PR TITLE
rpcserver: searchrawtx - update coinbase output

### PR DESCRIPTION
**Pull request by tuxcanfly**
Refs #186 

Added `Amount` field in coinbase output.

Also,  assuming that the fields - `BlockIndex` `BlockHeight` `tree` don't belong in coinbase output, I've updated the help text to reflect that.

### CODE REVIEW

**Comment by alexlyp** [comment]
tACK changes seem to work for me.  Will wait to merge until davec has verified that this adequately fixes the issue.

File: rpcserver.go [hunk 1]
```diff
@@ -1394,10 +1394,13 @@ func createVinListPrevOut(s *rpcServer, mtx *wire.MsgTx, chainParams *chaincfg.P
...
 		}
 
-		var vinEntry dcrjson.VinPrevOut
 		txIn := mtx.TxIn[0]
-		vinEntry.Coinbase = hex.EncodeToString(txIn.SignatureScript)
-		vinEntry.Sequence = txIn.Sequence
+		amtCoin := dcrutil.Amount(txIn.ValueIn).ToCoin()
+		vinEntry := dcrjson.VinPrevOut{
+			Coinbase: hex.EncodeToString(txIn.SignatureScript),
+			AmountIn: &amtCoin,
```
Comment (davecgh) [inline rpcserver.go]: dcrjson.Float64(dcrutil.Amount(txIn.ValueIn).ToCoin())

**Comment by davecgh** [comment]
Other than my comment, it reads fine.

File: rpcserver.go (hunk 1 above)
Comment (tuxcanfly) [inline rpcserver.go]: Updated.

**Pull request merged**